composer.json

```diff
@@ -6,11 +6,9 @@
       "email": "noreply@pantheon.io"
     }
   ],
-  "require-dev": {
-    "behat/mink-selenium2-driver": "~1.1",
-    "behat/mink-extension": "~2.0",
-    "symfony/dependency-injection": "2.7.11",
+  "require": {
```

Comment:
I've changed `require-dev` to `require` because Behat is a requirement for this repo to fulfill it's purpose, beyond developing the repo itself. This change also means that plugin packages don't need to directly require Behat, they can get it from this repo: https://github.com/pantheon-systems/wp-native-php-sessions/pull/39/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L18